### ISSUE TITLE
Datetime offset overflow

### ISSUE THREAD

**Issue by haluzpav** [ramonhagenaars/jsons#10]
It is possible to get following anomaly:

```python
utc = datetime.datetime.now(tz=datetime.timezone.utc)
local = datetime.datetime.now()
print(jsons.dump(utc))    # >>> 2019-02-17T20:14:32Z
print(jsons.dump(local))  # >>> 2019-02-18T00:14:32-20:00
```

I set my timezone to `+04:00`, `jsons` interprets it as `-20:00`. This suggests I am a day in future.

It is caused by the way you calculate datetime offset

`hrs_delta = datetime.now().hour - gmtime().tm_hour`

**Comment by haluzpav** [comment]
It is possible to get local timezone by

```
datetime.datetime.now().astimezone().tzinfo
```

**Comment by ramonhagenaars** [comment]
Thank you haluzpav! This bug will be addressed in the upcoming next release.

**Comment by ramonhagenaars** [comment]
Fixed in [0.7.0](https://pypi.org/project/jsons/0.7.0/).
